src/features/showAnnotations.js

```diff
@@ -22,20 +22,18 @@ const showAnnotations = async (compareFileData) => {
     );
 
     fileLinesWithChangedFiles.forEach(({ file, lines }) => {
-      const linesToDisplay = lines.map((line) =>
-        Array.isArray(line) ? line.join("-") : line
-      );
-
-      const linesWord = linesToDisplay.length === 1 ? "line is" : "lines are";
-
-      const formattedLines = linesToDisplay.join(", ");
-
-      const message = `file=${file}::${formattedLines} ${linesWord} not covered with tests`;
-
-      // NOTE: consider an option to show lines directly by attaching 'line' param
```

Comment:
I don't understand the case of this comment. Under what conditions does it occur?

Reply:
I think this is more an issue for Node.js applications coverage reports, mentioned here https://github.com/barecheck/code-coverage-action/issues/40

I think it's great to remove this comment, I will check one more if we can get it fixed before running this action.

Reply:
OK. I'll revert back to comment.